### ISSUE TITLE
jfxNative problem with openJDK

### ISSUE THREAD

**Issue by giloutho** [FibreFoX/javafx-gradle-plugin#146]
I use this plugin for two years without problems with Oracle JDK (V 8). I want to  swtich to openJDK (v8). 

I began with Mac version no problems. 

Under Windows, build felt with jfxNative : java.lang.NoSuchMethodError: com.oracle.tools.packager.windows.WinAppBundler.lambda$copyMSVCDLLs$261(Ljava/nio/file/Path;)Z

Application folder with executables is correct, the problem occured for the installer exe.

Any idea ?

**Comment by FibreFoX** [comment]
Did you install the OpenJFX-part too?

**Comment by giloutho** [comment]
Yes... I forgot : in Application foler, the exe generated works perfectly. Error occured during installer generation. 

It's not that serious, I can use Innosetup to produce the installer

**Comment by zapek** [comment]
I have the same problem here but with the Oracle JDK (v8, 221)

**Comment by chaddaniels** [comment]
Same problem here using Oracle or Zulu 8u222.

**Comment by irn** [comment]
The same issue on Windows with Oracle Java 1.8 (231)

**Comment by Indigo6** [comment]
Windows Oracle Java 1.8 (231) too

**Comment by chaddaniels** [comment]
Not sure if this is the right way to work around this issue, but I wanted to share this just in case this helped anyone else.  I added this and am no longer hitting the issue:
```
jfx {
   ...
   usePatchedJFXAntLib = false
   ...
}
```

**Comment by FibreFoX** [comment]
@chaddaniels that's a good explanation, on Windows there was some low-level binary-code patching required due to some bug, seems that they backported a lot of changes there :(

therefor: confirmed bug

**Comment by UgmaDevelopment** [comment]
Doing as @chaddaniels suggests worked for me, but make sure you stop your gradle daemon with something like `gradlew --stop` before trying again. 

There's a note that appears, reminding you to, but you may have missed it (like I did). It reads,
> Please restart gradle-daemon! Patched ant-javafx.jar is loaded, but you disabled to patch and use that file.

If you don't restart your daemon, you might get an error like this: 
```
Execution failed for task ':jfxNative'.
> java.lang.NoSuchMethodError:
```
